syntax = "proto3";

message Empty {}

message ShowtimeData {
    string date = 1;
    repeated string movies = 2;
}